doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -365,4 +365,7 @@ cJSON
 Documentation
 =============
 
-* Added a page on :ref:`ug_wireless_coexistence` in :ref:`protocols`.
+Added:
+
+* A page on :ref:`ug_wireless_coexistence` in :ref:`protocols`.
+* Pages on :ref:`thread_device_types` and :ref:`thread_sed_ssed` to the :ref:`ug_thread` documentation.
```

Comment:
I think this could be featured in Protocols/Thread section, actually.

Reply:
I was kind of 50/50 on it, seeing as it's purely a documentation addition.